packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss

```diff
@@ -83,11 +94,13 @@
   );
   @include add-alert-icon($icon, $banner-text-color-token, $bgcolor);
   @include set-text-and-bg($bgcolor);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: calc(units($theme-alert-icon-size) + $alert-icon-top-align * 2);
 
-  padding-left: units($theme-alert-icon-size) + ($alert-icon-optical-padding);
-  @include at-media($theme-site-margins-breakpoint) {
-    padding-left: units($theme-site-margins-width) * 2;
-    padding-right: units($theme-site-margins-width) * 2;
+  > * {
+    margin-left: $alert-x-align-text-with-icon;
```

Comment:
Update: Because the variable was no longer reused anywhere, I went ahead and removed the variable and just defined the value directly in the style rule. 

Additionally, I updated a lot of the methods for horizontal alignment, but think I want to keep this as is instead of moving it to body. The body is no longer oriented around just the text but the icon as well. To me, this is clearer and easier to work with but let me know if you want something different. Happy to discuss!